tests/sidetrail/working/patches/hmac.patch

```diff
@@ -9,71 +9,96 @@ index 3405781..3beeacf 100644
 +#include "smack.h"
 +#include "sidetrail.h"
 +
- int s2n_hmac_hash_alg(s2n_hmac_algorithm hmac_alg, s2n_hash_algorithm *out)
+ #include <stdint.h>
+
+ int s2n_hash_hmac_alg(s2n_hash_algorithm hash_alg, s2n_hmac_algorithm *out)
+@@ -191,10 +194,10 @@ int s2n_hmac_new(struct s2n_hmac_state *state)
+ S2N_RESULT s2n_hmac_state_validate(struct s2n_hmac_state *state)
  {
-     switch(hmac_alg) {
-@@ -235,7 +238,7 @@ int s2n_hmac_update(struct s2n_hmac_state *state, const void *in, uint32_t size)
+     ENSURE_REF(state);
+-    GUARD_RESULT(s2n_hash_state_validate(&state->inner));
+-    GUARD_RESULT(s2n_hash_state_validate(&state->inner_just_key));
+-    GUARD_RESULT(s2n_hash_state_validate(&state->outer));
+-    GUARD_RESULT(s2n_hash_state_validate(&state->outer_just_key));
++    //GUARD_RESULT(s2n_hash_state_validate(&state->inner));
++    //GUARD_RESULT(s2n_hash_state_valiLdate(&state->inner_just_key));
```

Comment:
Weirdest Nit: random "L" in the middle of "validate"?

Reply:
Fixed this typo.